data/README.md

```diff
@@ -15,7 +15,8 @@ En cas de problème vous pouvez redémarrer de zéro en faisant d'abord un `make
 ## Autres commandes :
 
 * `make image` : pour construire l'image docker choisie
-* `make import_agribalyse` : pour importer la base dans Brightway. Assurez-vous d'avoir le fichier `agribalyse3_no_param.CSV.zip` dans le dossier `food/import_agb/`
+* `make import_agribalyse` : pour importer Agribalyse 3.0 dans Brightway. Assurez-vous d'avoir le fichier `agribalyse3_no_param.CSV.zip` dans le dossier `food/import_agb/`
+* `make import_ei391_agb311` : pour importer Ecoinvent 3.9.1 et Agribaylyse 3.1.1. Assurez-vous d'avoir les bons fichiers dans dans le dossier `data/`
```

Comment:
This command doesn't exist in the Makefile. Et aussi, si elle devait d'aventure exister, il serait opportun de bien préciser quels fichiers sont nécessaires dans le dossier data, et comment les obtenir.

Trying `import_ecoinvent`, which exists and should probably be documented here, I'm getting this error:

```
Using environment variable BRIGHTWAY2_DIR for data directory:
/home/jovyan/data
### biosphere3 database is already imported
### Creating default LCIA methods
Traceback (most recent call last):
  File "/home/jovyan/ecobalyse/data/import_ecoinvent.py", line 60, in <module>
    main()
  File "/home/jovyan/ecobalyse/data/import_ecoinvent.py", line 41, in main
    bw2io.create_default_lcia_methods()
  File "/opt/conda/lib/python3.10/site-packages/bw2io/__init__.py", line 122, in create_default_lcia_methods
    ei = LCIAImporter("lcia_39_ecoinvent.zip")
  File "/opt/conda/lib/python3.10/site-packages/bw2io/importers/base_lcia.py", line 27, in __init__
    raise ValueError(
ValueError: Can't find biosphere database Agribalyse 3.1.1 biosphere
```